src/components/app/details/triggerView/EmptyStateCIMaterial.tsx

```diff
@@ -31,14 +34,35 @@ export default function EmptyStateCIMaterial({
                         className="empty-state__img--ci-material"
                     />
                 ),
-                title: <h1 className="dc__empty-title">{isRepoError ? repoErrorMsg : branchErrorMsg}</h1>,
+                title: <h1 className="dc__empty-title">{repoErrorMsg}</h1>,
                 subtitle: (
                     <a href={repoUrl} rel="noopener noreferrer" target="_blank">
                         {repoUrl}
                     </a>
                 ),
                 cta: null,
             }
+        } else if (isBranchError) {
+            const notConfiguredError = SOURCE_NOT_CONFIGURED_MESSAGE
+            return {
+                img: <img src={ErrorImage} alt="no commits found" className="empty-state__img--ci-material" />,
+                title: <h1 className="dc__empty-title">{branchErrorMsg}</h1>,
+                subtitle: repoUrl ? (
+                    <a href={repoUrl} rel="noopener noreferrer" target="_blank" className="">
+                        {repoUrl}
+                    </a>
+                ) : (
+                    <h1 className="dc__empty-title fs-13" style={{ color: 'gray' }}>
+                        {notConfiguredError}
```

Comment:
we can directly use `SOURCE_NOT_CONFIGURED_MESSAGE`

Reply:
done